src/test/compile-fail/keyword-super-as-identifier.rs

```diff
@@ -9,5 +9,7 @@
 // except according to those terms.
 
 fn main() {
-    let super = "foo"; //~ ERROR failed to resolve. There are too many initial `super`s
+    let super = "foo";
```

Comment:
let's change this test from `"foo"` to `22` -- I feel like the message about non-reference patterns isn't central to the test.